src/components/Buttons/button.css

```diff
@@ -1,36 +1,29 @@
 /** @format */
 
-.category-btn {
+.option-btn {
   all: unset;
-  padding: 0.4rem 1rem;
-  border-radius: 40px;
+  padding: 1rem;
+  border-radius: 10px;
   font-weight: 600;
   border: 2px solid var(--primary-color);
   cursor: pointer;
+  width: 28rem;
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  color: black;
+  transition: 200ms;
```

Comment:
Suggestion: You could separate structure from the style within a CSS rule and group them for ease of maintenance.

Reply:
will do that